app/views/searches/advanced.html.erb

```diff
@@ -1,8 +1,16 @@
 <% @title = t("advanced_search") %>
 
-<div class="home__search-wrap" role="search">
-  <%= form_tag search_url, id: "advanced-search", method: :get do %>
-    <%= search_field_tag :query, params[:query], placeholder: t("layouts.application.header.search_gem_html"), autofocus: current_page?(root_url), id: "query", class: "home__search" %>
+<div class="home__search-wrap" role="search" data-controller="search">
+  <%= form_tag search_url, id: "advanced-search", method: :get, data: { search_target: "form" } do %>
```

Comment:
Is this data attribute still used? 🤔

Reply:
oh good catch, I'll remove it. I accessed the form through queryTarget.form instead.